assets/app/scripts/controllers/builds.js

```diff
@@ -50,6 +52,12 @@ angular.module('openshiftConsole')
         }
       }
 
+      // Scroll to anchor on first load if location has a hash.
+      if (!action && $location.hash()) {
+        // Wait until the digest loop completes.
+        setTimeout($anchorScroll, 10);
```

Comment:
OK

On Tue, Aug 11, 2015 at 3:23 PM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/scripts/controllers/builds.js
> <https://github.com/openshift/origin/pull/3406#discussion_r36787157>:
>
> > @@ -50,6 +52,12 @@ angular.module('openshiftConsole')
> >          }
> >        }
> >
> > +      // Scroll to anchor on first load if location has a hash.
> > +      if (!action && $location.hash()) {
> > +        // Wait until the digest loop completes.
> > +        setTimeout($anchorScroll, 10);
>
> We're not changing any data, so no need to trigger a digest cycle. We had
> some performance issues with $timeout and $interval and hawtio-nav, so I
> try to avoid them now if not necessary.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3406/files#r36787157>.
>